pandas/io/tests/test_excel.py

```diff
@@ -780,7 +792,7 @@ def test_tsframe(self):
         with ensure_clean(self.ext) as path:
             df.to_excel(path, 'test1')
             reader = ExcelFile(path)
```

Comment:
in most of these just dispense with using ``ExcelFile`` entirely (leave 1 or 2 out of all of the tests for compat though)